media/jsonp/server/pom.xml

```diff
@@ -45,13 +44,18 @@
             <version>${project.version}</version>
         </dependency>
         <dependency>
-            <groupId>org.glassfish</groupId>
-            <artifactId>javax.json</artifactId>
-        </dependency>
-        <dependency>
-            <groupId>io.projectreactor</groupId>
-            <artifactId>reactor-core</artifactId>
+            <groupId>io.helidon.media.jsonp</groupId>
+            <artifactId>helidon-media-jsonp-common</artifactId>
+            <version>${project.version}</version>
         </dependency>
+        <!--<dependency>-->
```

Comment:
Why leaving these commented out rather than deleted them ?

Reply:
It is a leftover when I wanted to try if these dependencies are needed. Forgot to delete them, will do that.